chapters/operatorsandexpressions.tex

```diff
@@ -438,10 +436,10 @@ \subsection{Numeric Functions and Conversion Functions}\doublelabel{numeric-func
 
 Format string: According to ANSI-C the format string specifies one
 conversion specifier (excluding the leading \%), may not contain length
-modifiers, and may not use "\lstinline!*!" for width and/or precision. For~all
-numeric values~the~format specifiers~\lstinline!f!, \lstinline!e!, \lstinline!E!, \lstinline!g!,\lstinline!G! are allowed. For
-integral values~it is also allowed to~use the~d, i, o, x, X, u, and
-c-format specifiers (for non-integral values a tool may~round, truncate
+modifiers, and may not use `\lstinline!*!' for width and/or precision. For all
+numeric values the format specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed. For
```

Comment:
Similarly as the previous comment I think this should be either be \lstinline!f! without quotes, or \lstinline!"f"! (etc)

Reply:
Right, here the single quotes are mostly added for consistency, they are probably not really needed for separation from the surrounding text.

Other places that could help us find a good way to write the format specifiers could be where `printf` is documented, such as:
- http://www.cplusplus.com/reference/cstdio/printf/
- https://docs.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=vs-2019
- https://en.cppreference.com/w/cpp/io/c/fprintf

None of these sources write the format specifier as a complete (double-quoted) string literal.

Reply:
True, it seems they (and the C-standard) just use the format, but formatted as code.